rega-genotype/src/main/java/rega/genotype/tools/FastaToRega.java

```diff
@@ -43,21 +48,30 @@
 	}
 
 	public static void main(String[] args) throws Exception {
-
+		
```

Comment:
Please remove all the changes to the whitespaces that have no effect on this file